dbx/commands/configure.py

```diff
@@ -52,17 +52,32 @@ def configure(
         "--enable-context-based-upload-for-execute",
         is_flag=True,
         help="""
-        Enables failsafe behaviour for assets-based launches with definitions
-        that are based on shared job clusters feature.
+        Enables context based uploader instead of MLflow uploader.
 
         This flag ignores any other flags.
 
 
+        Project file should exist, otherwise command will fail.""",
+    ),
+    enable_custom_init_scripts: bool = typer.Option(
+        False,
+        "--enable-custom-init-scripts",
+        help="""Enables the merge of the init scripts from the cluster policy and
+        those from the `new_cluster.init_scripts` key.
+
+        This flag can coexist with other flags.
+
+
         Project file should exist, otherwise command will fail.""",
     ),
 ):
     manager = ProjectConfigurationManager()
 
+    if enable_custom_init_scripts:
+        dbx_echo("Enabling custom init scripts")
+        manager.enable_custom_init_scripts()
+        dbx_echo("✅ Enabling custom init scripts")
+
     if enable_inplace_jinja_support:
```

Comment:
it should be a chain of if/elif, not if-if-if.

Reply:
@renardeinside 
Indeed I haven't done a full test of all the parameters combinations, but the new switch `--enable-custom-init-scripts` is designed to be able to coexist with other switches, that's why I used `if-if-elif-else`, otherwise I can not use  `--enable-custom-init-scripts` with `--enable-inplace-jinja-support` together.

![image](https://user-images.githubusercontent.com/13083856/198077173-e717ffaf-ce92-4267-9150-6d458d662ac2.png)

Reply:
@renardeinside 
correct me if my understanding is wrong

Reply:
the new switch `append-init-scripts` is handled in `manager.create_or_update()` now